ProjectMarsLanguageFeature/StepDefinitions/LanguageAddStepDefinitions.cs

```diff
@@ -13,7 +14,7 @@ public class LanguageAddStepDefinitions : CommonDriver
         LanguagesPage languagesPageObj = new LanguagesPage();
         DeleteAnExistingLanguage deletingObj = new DeleteAnExistingLanguage();
         ContentReadLanguage readLanguageObj = new ContentReadLanguage();
-       
+
         [When(@"I click on Add New buttons")]
```

Comment:
have you thought about state
so example before adding language you need to make sure there a no existing languages as this could result in false positive test pass
you will need to think about this for all the scenarios

the state is say that you have a test that is adding a language english, the second time you run this test you should delete any existing languages so that the state is reset, this will guarantee that your test does not produce false negatives and passes always

Reply:
I thought of state, for that in test sequence I put Delete function to run before add function so that all the existing languages will delete, but I think I should make it specifically for add function so it can run independent .